Client/Frontend/Settings/ThemeSettings/ThemeSettingsState.swift

```diff
@@ -2,90 +2,77 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
+import Common
 import Redux
 
-enum ThemePicker: Equatable {
-    case light
-    case dark
-}
-
-enum SwitchMode: Equatable {
-    case manual(ThemePicker)
-    case automatic(Float)
-}
+protocol ReduxState {}
```

Comment:
Comment for this `ReduxState` protocol? And we should probably put it in another file